screens.json

```diff
@@ -332,6 +332,12 @@
 		"dppx": 2
 	},
 	{
+		"name": "Google Pixel C",
+		"w": 2560,
+		"h": 1800,
+		"ppi": 308,
```

Comment:
No diagonal? Also, are you sure ppi is needed? It's automatically calculated, we only include it when it's different than the automatic one.